static/js/test_setup.ts

```diff
@@ -44,6 +44,7 @@ declare global {
 // cleanup after each test run
 // eslint-disable-next-line mocha/no-top-level-hooks
 afterEach(function() {
+  jest.resetAllMocks()
```

Comment:
This change will apply to all our jest mocks... But the tests still pass, so it seems like something we should do.

Reply:
makes sense I think, I can't really think of a case where we'd want mocks sticking around between tests